react-app/src/components/users/UsersTable.tsx

```diff
@@ -59,6 +60,14 @@ interface IUsersTable {
   error: unknown;
 }
 
+const statusComparitor: GridComparatorFn = (v1, v2) => {
```

Comment:
This works but it would be better if it indicated the intention of this sort more clearly at a glance. Adding a comment explaining as much could be sufficient. Though I will suggest one alternative approach:

```
const statusOrder = ['OnHold', 'Active', 'Disabled', 'Denied'];
const indx1 = statusOrder.indexOf(v1);
const indx2 = statusOrder.indexOf(v2);
if (indx1 < 0 || indx2 < 0) return 0;
else return indx2 - indx1;
```
This way you can see that the sort order is clearly demonstrated by the order in which elements appear within the array. The if statement at the bottom is just to guard against any unknown values appearing (indexOf returns -1 on failure).

Reply:
I like this a lot. thanks for the help!